BLE.Client/BLE.Client/BleMvxApplication.cs

```diff
@@ -7,7 +7,7 @@ public class BleMvxApplication : MvxApplication
     {
         public override void Initialize()
         {
-			RegisterAppStart<SetupViewModel>();
+            RegisterAppStart<LoginViewModel>();
```

Comment:
not sure we should put this in master until we figure out the logic for deciding between login view model and setup view model...